dsc/registryResource.md

```diff
@@ -27,34 +26,26 @@ Registry [string] #ResourceName
 ```
 
 ## Properties
+
 |  Property  |  Description   |
 |---|---|
 | Key| Indicates the path of the registry key for which you want to ensure a specific state. This path must include the hive.|
 | ValueName| Indicates the name of the registry value. To add or remove a registry key, specify this property as an empty string without specifying ValueType or ValueData. To modify or remove the default value of a registry key, specify this property as an empty string while also specifying ValueType or ValueData.|
 | Ensure| Indicates if the key and value exist. To ensure that they do, set this property to "Present". To ensure that they do not exist, set the property to "Absent". The default value is "Present".|
-| Force| If the specified registry key is present, __Force__ overwrites it with the new value. If deleting a registry key with subkeys, this needs to be __$true__|
-| Hex| Indicates if data will be expressed in hexadecimal format. If specified, the DWORD/QWORD value data is presented in hexadecimal format. Not valid for other types. The default value is __$false__.|
-| DependsOn| Indicates that the configuration of another resource must run before this resource is configured. For example, if the ID of the resource configuration script block that you want to run first is __ResourceName__ and its type is __ResourceType__, the syntax for using this property is `DependsOn = "[ResourceType]ResourceName"`.|
+| Force| If the specified registry key is present, **Force** overwrites it with the new value. If deleting a registry key with subkeys, this needs to be **$true** |
+| Hex| Indicates if data will be expressed in hexadecimal format. If specified, the DWORD/QWORD value data is presented in hexadecimal format. Not valid for other types. The default value is **$false**.|
+| DependsOn| Indicates that the configuration of another resource must run before this resource is configured. For example, if the ID of the resource configuration script block that you want to run first is **ResourceName** and its type is **ResourceType**, the syntax for using this property is `DependsOn = "[ResourceType]ResourceName"`.|
 | ValueData| The data for the registry value.|
 | ValueType| Indicates the type of the value. The supported types are:
-<ul><li>String (REG_SZ)</li>
-
-
-<li>Binary (REG-BINARY)</li>
-
-
-<li>Dword 32-bit (REG_DWORD)</li>
-
-
-<li>Qword 64-bit (REG_QWORD)</li>
-
-
-<li>Multi-string (REG_MULTI_SZ)</li>
-
-
-<li>Expandable string (REG_EXPAND_SZ)</li></ul>
+- String (REG_SZ)
```

Comment:
This table is broken. You need to use the HTML markup for the list and close the table. Or turn the bullet list into a CSV list.